Aztec/Classes/TextKit/MediaAttachment.swift

```diff
@@ -49,9 +49,24 @@ open class MediaAttachment: NSTextAttachment {
     
     /// Attachment URL
     ///
-    open var url: URL?
+    open var url: URL? {
```

Comment:
Do we want this to be overridden?

Allowing this value to be overriden will most likely break the retry count, as the setter will never be executed.